app/importers/templates/activity-upgrade.php

```diff
@@ -0,0 +1,135 @@
+<?php
+/**
+ * Template for Activity Upgrade - RTMediaActivityUpgrade::init().
+ *
+ * @package rtMedia
+ */
+
+?>
+
+<div class="wrap">
+	<h2><?php esc_html_e( 'rtMedia: Upgrade rtMedia activity', 'buddypress-media' ); ?></h2>
+	<?php
+	wp_nonce_field( 'rtmedia_media_activity_upgrade_nonce', 'rtmedia_media_activity_upgrade_nonce' );
+
+	printf(
+	// translators: %2$s: Estimated.
```

Comment:
This comment not needed because there is no placeholder used in translatable string. So remove it.

Reply:
Done